readme.md

```diff
@@ -6,10 +6,10 @@ A real-time culture sharer using Firebase as its backend and AngularJS on the fr
 ## Installation
     
     # Install global NPM dependencies:
-    npm install -g gulp bower jshint recess
+    npm install -g gulp bower jshint recess yarn
```

Comment:
https://github.com/yarnpkg/yarn is not a npm package. Could you modify your pull request please ?